react/modules/gaEvents.ts

```diff
@@ -118,3 +119,36 @@ export function selectPromotion(eventData: PixelMessage['data']) {
 
   updateEcommerce(eventName, data)
 }
+
+export function removeFromCart(eventData: PixelMessage['data']) {
+  if (!shouldMergeUAEvents()) return
+
+  const eventName = 'remove_from_cart'
+
+  const {
+    currency,
+    items: [item],
```

Comment:
Similar to what I [mentioned](https://github.com/vtex-apps/google-tag-manager/pull/106#discussion_r1106328966) in this other PR, let's update this to handle the removal of multiple items. It appears [our current app that sends this event doesn't handle multiple items](https://github.com/vtex-apps/minicart/blob/6c5f3a02700fa18cc762a0c59a7a8d69f3ea634a/react/ProductList.tsx#L53-L69), but since [Google's spec](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?hl=en&client_type=gtag#remove_from_cart) allows multiple, we should support it to be more future-proof.